### PR TITLE
readme: minimal overview

### PR DESCRIPTION
**Pull request by hunchback**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,18 @@
-# ovsdb-etcd
+[![Build Status](https://travis-ci.com/IBM/ovsdb-etcd.svg?branch=master)](https://travis-ci.com/IBM/ovsdb-etcd "Travis")
+[![Go Report Card](https://goreportcard.com/badge/github.com/IBM/ovsdb-etcd)](https://goreportcard.com/report/github.com/IBM/ovsdb-etcd)
 
-A PoC project to store OVN database date into an etcd cluster   
\ No newline at end of file
+# OVSDB ETCD Backend (PoC)
+
+A PoC project to store OVN database date into an etcd cluster, as by doing so
+we can levarge the advantages of the widly used ETCD key-value store. As an
```
Comment (roytman) [inline README.md:7]: we can levarge the advantages of the widly used ETCD key-value store ->
we can leverage the advantages of the widely used distributed key-value store - etcd.

File: README.md [hunk 2]
```diff
@@ -1,3 +1,18 @@
...
+alternative to the propriety transactional store curently serving OVSDB. The
+goal is that this be a drop in replacement which is fully compatible with any
+of the existing ovsdb clients.
+
+You will find the spec here
+[rfc7047](https://tools.ietf.org/html/rfc7047#page-19),
+and you will find a description of the extentions beyond the spec here
```
Comment (roytman) [inline README.md:14]: extensions

File: README.md [hunk 3]
```diff
@@ -1,3 +1,18 @@
-# ovsdb-etcd
+[![Build Status](https://travis-ci.com/IBM/ovsdb-etcd.svg?branch=master)](https://travis-ci.com/IBM/ovsdb-etcd "Travis")
+[![Go Report Card](https://goreportcard.com/badge/github.com/IBM/ovsdb-etcd)](https://goreportcard.com/report/github.com/IBM/ovsdb-etcd)
 
-A PoC project to store OVN database date into an etcd cluster   
\ No newline at end of file
+# OVSDB ETCD Backend (PoC)
+
+A PoC project to store OVN database date into an etcd cluster, as by doing so
```
[the remark this answers is not in the document]
Comment (hunchback) [reply README.md:6]: ok

File: README.md (hunk 1 above)
Comment (hunchback) [reply README.md:7]: ok

**Pull request merged**